src/Gui/Viewer/CameraManipulator.cpp

```diff
@@ -71,6 +71,42 @@ void CameraManipulator::attachLight( Engine::Scene::Light* light ) {
 KeyMappingManager::Context Gui::CameraManipulator::mappingContext() {
     return Gui::KeyMappingManager::Context();
 }
+/// @return true if the event has been taken into account, false otherwise
```

Comment:
Why do we repeat doxygen comments is the cpp file ? (same for other functions)

Reply:
because of quick copy paste.